pkg/api/v1/router_server.go

```diff
@@ -100,6 +100,16 @@ func (r *Router) serverCreate(c *gin.Context) {
 		return
 	}
 
+	exists, err := models.ServerExists(c.Request.Context(), r.DB, srv.UUID.String())
```

Comment:
I thought the UUID was auto-generated.

Reply:
Nice find, I'll go test this separately again (now I wonder why it worked in my tests).

reverted commits as  e3cae69 23091ae